ui/src/admin/components/chronograf/ProvidersTable.js

```diff
@@ -0,0 +1,149 @@
+    this.setState({isCreatingMap: false})
+  }
+
+  render() {
+    const {
+      mappings = [],
+      organizations,
+      onUpdateMap,
+      onDeleteMap,
+      isLoading,
+    } = this.props
+    const {isCreatingMap} = this.state
+
+    const tableTitle =
+      mappings.length === 1 ? '1 Map' : `${mappings.length} Maps`
+
+    // define scheme options
+    const SCHEMES = [{text: '*'}, {text: 'oauth2'}]
+
+    if (isLoading) {
+      return (
+        <div className="panel panel-default">
+          <div className="panel-body">
+            <div className="page-spinner" />
+          </div>
+        </div>
+      )
+    }
+
+    return (
+      <div className="panel panel-default">
+        <div className="panel-heading u-flex u-ai-center u-jc-space-between">
+          <h2 className="panel-title">
+            {tableTitle}
+          </h2>
+          <button
+            className="btn btn-sm btn-primary"
+            onClick={this.handleClickCreateMap}
+            disabled={isCreatingMap}
+          >
+            <span className="icon plus" /> Create Mapping
+          </button>
+        </div>
+        {mappings.length || isCreatingMap
+          ? <div className="panel-body">
+              <div className="fancytable--labels">
+                <div className="fancytable--th provider--scheme">Scheme</div>
+                <div className="fancytable--th provider--provider">
+                  Provider
+                </div>
+                <div className="fancytable--th provider--providerorg">
+                  Provider Org
+                </div>
+                <div className="fancytable--th provider--arrow" />
+                <div className="fancytable--th provider--redirect">
+                  Organization
+                </div>
+                <div className="fancytable--th" />
+                <div className="fancytable--th provider--delete" />
+              </div>
+              {mappings.map((mapping, i) =>
+                <ProvidersTableRow
+                  key={uuid.v4()}
+                  mapping={mapping}
+                  organizations={organizations}
+                  schemes={SCHEMES}
+                  onDelete={onDeleteMap}
+                  onUpdate={onUpdateMap}
+                  rowIndex={i + 1}
+                />
+              )}
+              {isCreatingMap
+                ? <ProvidersTableRowNew
+                    organizations={organizations}
+                    schemes={SCHEMES}
+                    onCreate={this.handleCreateMap}
+                    onCancel={this.handleCancelCreateMap}
+                    rowIndex={mappings.length + 1}
+                  />
+                : null}
+            </div>
+          : <div className="panel-body">
+              <div className="generic-empty-state">
+                <h4 style={{margin: '50px 0'}}>
+                  Looks like you have no mappings<br />
+                  New users will not be able to sign in
```

Comment:
This copy is outside of the scope of my review, but I think it would be better as `New users will not be able to sign in automatically? `